source/3d/onshape.md

```diff
@@ -0,0 +1,11 @@
+# Modifying the source design using onshape
+
+The design was created using [onshape](https://www.onshape.com/en/) which is a cloud based cad platform.
+
+The current version of the design can be accessed [here](https://cad.onshape.com/documents/4a734179a4bf72ca3cf4dc82/v/0fcfc861c56588e719f614ad/e/0323c8faa4d42ca9fcdb2b1a?renderMode=0&uiState=628bbdd0a1042f44aeda13e6)
```

Comment:
https://cad.onshape.com/documents/34e52f69664fd15550ba35ad/w/5abb2d265c5abd27c227e7a7/e/b3a911c86048cfb278f7aa6c?renderMode=0&uiState=62cf7d917c0f5407a2eca93e 

New proposed design. 
Changes:
- Thickness = 2.4mm (easier to print with a 0.8mm nozzle)
- Added M3 screw holes for the C13/C14 connectors
- Added snap-on tab underneath the USB connector for extra support
- Added chamfer to the existing standoffs to improve adhesion with the bottom plate

Reply:
> There isn't much benefit in having the opening in both top and bottom part. If you push your lines you could keep all openings in the on part and have the other just restricting the open end. Also, by doing this you could use snap on C13/C14 modules while having the case open.

Okay, will update to split the case at the top instead of the middle

> Btw, why are we using screw on panel mount?

I added the screw mounts as C13/C14 are much easier to source here in Brazil and they're compatible with the snap on ones. I figured that it'd be nice to leave the case open to alternatives.

> Better remove this overhand to avoid drama in printing, There's no harm to have this section going all the way down so it's self supported. Also, no need to use the two small pins since the port opening restricts the USB module in 2 dimensions.

Okay, will simplify this part.

> Here you can avoid using screws. Instead of 2 pillars, you could have a rect pillar on each end of the case that restricts the USB module from moving up/down or going back/ forth

Nice idea, I'll try to implement it and add it to the next revision.

> Better use max 2mm radius on the fillet so it prints nicely without support. The adjacent edge can remain 5mm or it could even be larger.

Noted.

Reply:
![photo_5093862957615524359_y](https://user-images.githubusercontent.com/4000637/179638601-76324953-edc4-4489-b8f2-8df5d495f8a0.jpg)

I've prototyped the bottom part already, the USB Relay fits very snugly. The rectangular pillars were a little bit too short, so I've added an extra 0.5mm to the model. Besides that, it all went great.

Reply:
![IMG_20220721_204942](https://user-images.githubusercontent.com/4000637/180496783-6fe168a7-8c2c-4311-9899-d7d126daa9bb.jpg)
![IMG_20220721_210055](https://user-images.githubusercontent.com/4000637/180496793-4d23ce6a-9b84-4559-9236-2a6b0baf8b25.jpg)
![IMG_20220721_210102](https://user-images.githubusercontent.com/4000637/180496797-6e176e02-3603-4e83-acf5-46696bcaced1.jpg)

Reply:
Final assembled and tested unit, worked out great.

Notes:

- Maybe it'd be interesting to consolidate all screws to M2.5, instead of using both M3 and M2.5. I'll consider this a stretch goal.
- I've made the top lid much thinner, from ~5mm to 2.4mm. This decreased print time dramatically while retaining strength.
- Wiring was tight with 1.5mm^2 wires, but doable. The enclosure should not be reduced further without considering the bending radius.
- Brown wire was near impossible to find, so I used red in my assembly. Further builds should use brown to respect IEC code.